packages/meditrak-server/package.json

```diff
@@ -34,7 +34,6 @@
     "async": "^2.6.2",
     "aws-sdk": "^2.451.0",
     "body-parser": "^1.18.3",
-    "cached": "^4.3.2",
```

Comment:
The user reward and leaderboard endpoints previously had their sql query results cached, I guess as a performance optimisation, but:
- the cache was only valid for 20 seconds before being marked stale
- the queries themselves (or the new ones at least) are super quick, so could be premature optimisation
- in practise with the cache removed I haven't seen any performance degradation
- it was one of many cache layers/strategies we have around the place, so thought it was better to delete it to remove a point of complexity